### PR TITLE
[CYS on Core] Update homepage templates intro patterns

### LINKED ISSUE #44614: [CYS on Core] Update homepage templates intro patterns
We need to update the homepage layouts on Core to be aligned with the proposed designs here:
pdnLyh-5n3-p2

### PR DESCRIPTION
**Pull request by albarin**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

This PR updates the intro patterns for Core in the first and second homepages of the CYS flow.
It also adds bottom margin to the `Just Arrived Full Hero` and increases the height of the `Featured Category Cover Image` to match the designs.

Closes https://github.com/woocommerce/woocommerce/issues/44614

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

**Local/JN**
1. Ensure the `WooCommerce Beta Tester` plugin is installed and activated (available on this monorepo).
2. Head over to `/wp-admin/tools.php?page=woocommerce-admin-test-helper` and enable `customize-store` feature flag.
3. Visit the `wp-admin/admin.php?page=wc-admin&path=/customize-store`.
4. Click on `Customize your store` and finish the flow.
5. Check the first pattern after the header is the `Just Arrived Full Hero` 👇 
Screenshot 2024-02-14 at 16 15 12

7. Click on `Change your homepage` and click on the second homepage available.

8. Check the first pattern after the header is the `Featured Category Cover Image` 👇 
Screenshot 2024-02-14 at 16 15 42

9. Click on the third homepage option.
10. Check the first pattern after the header is the `Hero Product Split` 👇
Screenshot 2024-02-14 at 16 40 54

**WooExpress**
1. Ensure the `WooCommerce Beta Tester` plugin is installed and activated (available on this monorepo).
2. Head over to `/wp-admin/tools.php?page=woocommerce-admin-test-helper` and enable `customize-store` feature flag.
3. Visit the `wp-admin/admin.php?page=wc-admin&path=/customize-store`.
4. Click on `Customize your store` and finish the flow.
5. Click on `Change your homepage`.
6. Check that in the 3 available homepages the first pattern after the header is the `Hero Product Split` 👇
Screenshot 2024-02-14 at 16 40 54

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce-admin/client/customize-store/data/homepageTemplates.ts [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
 	template1: {
 		blocks: [
 			// Header
 			'woocommerce-blocks/header-centered-menu',
 
 			// Body
-			'woocommerce-blocks/hero-product-split',
+			isWooExpress()
+				? introPatternWooExpress
+				: 'woocommerce-blocks/just-arrived-full-hero',
```
Comment (nefeline) [inline plugins/woocommerce-admin/client/customize-store/data/homepageTemplates.ts:17]: @albarin do you mind confirming what's the expected conditional here, please? ATM the `'woocommerce-blocks/just-arrived-full-hero'` is being used both on Woo Core and on WooExpress (`const introPatternWooExpress = 'woocommerce-blocks/just-arrived-full-hero';`)

PS: if the decision is to have `'woocommerce-blocks/just-arrived-full-hero'` on WooExpress, we also have to update the following constant to ensure AI content is generated for it: https://github.com/woocommerce/woocommerce/blob/9d86e1f038a91e190ef9f53d1042ec2d04ff633d/plugins/woocommerce/src/Blocks/AIContent/UpdatePatterns.php#L16-L26

**Review by nefeline — CHANGES_REQUESTED** [review]
Nice work!

Things are a-ok on core, but on WooExpress I've noticed some differences: left a comment to confirm 👍

File: plugins/woocommerce-admin/client/customize-store/data/homepageTemplates.ts (hunk 1 above)
Comment (albarin) [reply plugins/woocommerce-admin/client/customize-store/data/homepageTemplates.ts:17]: My bad, you are totally right, the idea is to keep the `woocommerce-blocks/hero-product-3-split` as intro in WooExpress. Just pushed a fix 🙏

**Review by nefeline — APPROVED** [review]
LGTM 🚀 !

**Pull request merged**